src/vs/workbench/contrib/terminalContrib/suggest/test/browser/terminalCompletionService.test.ts

```diff
@@ -157,27 +157,5 @@ suite('TerminalCompletionService', () => {
 				replacementLength: 3 // replacing ./f
 			});
 		});
-		test('cd ../', async () => {
-			const resourceRequestConfig: TerminalResourceRequestConfig = {
-				cwd: URI.parse('file:///folder1'),
-				foldersRequested: true,
-				pathSeparator
-			};
-			await fileService.createFolder(URI.parse('file:///test/'));
```

Comment:
I think you'll want to mock fileService here instead as in don't interact with the OS at all and instead have it return true for a fixed set of resources. That's probably why it's failing in CI as it depends upon the file system.

Something closer to this:

https://github.com/microsoft/vscode/blob/ae4fcd92ba98d58b936c58ac6a36e4a05af14980/src/vs/workbench/contrib/terminalContrib/links/test/browser/terminalLocalLinkDetector.test.ts#L203-L210

Don't want to introduce another flaky test 😅

Reply:
done